src/app/services/graph-client/msal-service.ts

```diff
@@ -1,101 +1,31 @@
-import { AuthenticationParameters } from 'msal';
```

Comment:
how much work is it to completely get rid of this file and use AuthenticationModule directly.  The functions here are pretty thin so I don't see a ton of value other than less change.

Reply:
I have moved the functions there to an authentication wrapper that would handle our interactions with authentication from the application